README.md

```diff
@@ -148,16 +148,12 @@ func (s *server) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
         panic("TODO: handle error")
     }
 
+    // Set the response headers
```

Comment:
Not a blocker for this PR, but we discussed potentially creating an example `_test.go` file.

DO you think it's still worth having the code in the README instead?

Reply:
The reason I was reluctant to add it as a test, is that it'll have to demonstrate how it's used within an `http` server.

I'll do it in this PR, if you don't think it's not too much for an example test :+1:

Reply:
Let's do it

Reply:
@Olshansk , after an initial attempt it seems that to have a working test, we need to mock the whole dependencies (mainly connectivity to the blockchain), a supplier server and an app gate server.

Pushing this back for now and creating a test ticket.

Reply:
Can we #PUC w/ a TODO + link to the ticket in the code?